pages/features/list.md

```diff
@@ -144,7 +144,7 @@ In a perfectly balanced 5-player party, each player should get 20%.
 
 Calls out players who may be slacking in dungeons, using the Dungeon Player Rater. Uses a configurable automatated message. 
 
-![image](https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/5e1fa3f2-699d-4812-b757-867bcd9f4da7)
+![image](/images/dungeon_snicher.png)
```

Comment:
This one seems to be broken, maybe it was a diff one

Reply:
Typo should say snitched